src/infra/core/src/utils/simple_transfer_protocol.rs

```diff
@@ -19,13 +20,39 @@ use crate::*;
 
 /////////////////////////////////////////////////////////////////////////////////////////
 
+const ENV_VAR_SIMPLE_PROTOCOL_MAX_PARALLEL_TRANSFERS: &str =
+    "SIMPLE_PROTOCOL_MAX_PARALLEL_TRANSFERS";
+const DEFAULT_SIMPLE_PROTOCOL_MAX_PARALLEL_TRANSFERS: usize = 10;
```

Comment:
What is the motivation for exactly 10 ?

Reply:
It is debatable, but for me, it is a good number for parallel requests that will not spam any server.